core/src/main/java/org/elasticsearch/ElasticsearchCorruptionException.java

```diff
@@ -42,10 +42,7 @@ public ElasticsearchCorruptionException(String message) {
      * @param ex the exception cause
      */
     public ElasticsearchCorruptionException(Throwable ex) {
-        this(ex.getMessage());
-        if (ex != null) {
-            this.setStackTrace(ex.getStackTrace());
-        }
+        super(ex);
         Throwable[] suppressed = ex.getSuppressed();
```

Comment:
Didn't see the suppressed stuff being filled in when looking at the implementation of Throwable(Throwable cause) and Throwable.fillInStackTrace(...) that's why I left it there. More than happy to remove even more lines though.